src/prefect/prefect.yaml

```
@@ -11,9 +15,11 @@ deployments:
         timezone: "Europe/Paris"
         active: true
     parameters:
+      environment: development
+      levels: [0, 1]
```

Scheduled deployments should be configured for production:

```suggestion
      environment: production
      levels: [0, 1, 2, 3, 4]
```